src/components/ProgressPage.tsx

```diff
@@ -0,0 +1,90 @@
+import { FC } from "react";
+import { Box, Stepper, Typography, Step, StepLabel} from "@mui/material";
+import { TestPhase } from "../contexts/general.context";
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+import RadioButtonUncheckedIcon from '@mui/icons-material/RadioButtonUnchecked';
+
+
+const MemoryTestsLabels = [
+    {
+        "name": "BoCA - Memory (Immediate Recall)",
+        "desc": "BoCA description",
+        "index": TestPhase.MEMORY_RECALL_IMMEDIATE
+    },
+    {
+        "name": "TestMyBrain - Visual Paired Associates (\"learn\" part)",
+        "desc": "VPA desc",
+        "index": TestPhase.VISUAL_PAIRS_MEMORIZE
+    },
+    {
+        "name": "TestMyBrain - Choice Reaction Time (from Visual Paired Associates)",
+        "desc": "CRT desc",
+        "index": TestPhase.CHOICE_REACTION_TIME
+    },
+    {
+        "name": "TestMyBrain - Visual Paired Associates (\"test\" part)",
+        "desc": "DSM desc",
+        "index": TestPhase.VISUAL_PAIRS_RECALL
+    },
+    {
+        "name": "TestMyBrain - Digit Symbol Matching",
+        "desc": "Spatial Memory desc",
+        "index": TestPhase.DIGIT_SYMBOL_MATCHING
+    },
+    {
+        "name": "NeurCare - Spatial Memory",
+        "desc": "BoCA delayed recall desc",
+        "index": TestPhase.SPACIAL_MEMORY
+    },
+    {
+        "name": "BoCA - Memory (Delayed recall)",
+        "desc": "BoCA delayed recall desc",
+        "index": TestPhase.MEMORY_RECALL_DELAYED
+    }
+];
+
+
+interface ProgressPageProps  {  
+    id:number
+}
+
+export const ProgressPage: FC<ProgressPageProps> = (props) => {
+
+    //check for Dark/Light Mode
+    const  dark_light_color = () => {
+        if(window.matchMedia('(prefers-color-scheme: dark').matches){
```

Comment:
Maybe use double quotation mark instead. It would be better to keep consistency.